theme/index.ts

```diff
@@ -1,2 +1,13 @@
-export * from './ThemeProvider';
-export * from './Theme';
+import { Color, RGB } from './colors';
+import { ThemeProvider } from './ThemeProvider';
+import { Theme as ThemeImpl } from './Theme';
+
+// Adding statics so that we can export a module that is useable as:
+// <Theme>
+// <Theme.Provider>
+// etc
+const Theme = Object.assign({}, ThemeImpl) as any; // any is so we can add the random statics below
+Theme.Provider = ThemeProvider;
+
+export { Theme, Color, RGB };
+export type VarnishTheme = typeof ThemeImpl;
```

Comment:
```suggestion
import { Theme } from './Theme';

export {
   Provider: ThemeProvider,
   RGB,
   Color,
   ...Theme
}
```

I think you can do ☝️  to avoid the `any` n' all that.

Reply:
done